__tests__/app.test.js

```diff
@@ -39,14 +39,66 @@ describe("GET /api/topics", () => {
   });
 });
 
-describe('GET /api', () => {
-    it('should return an object detailing all the available endpoints on the API', () => {
-        return request(app)
-        .get("/api")
-        .expect(200)
-        .then((data) => {
-            const sampleObj = allEndpoints
-            expect(data.body).toEqual(sampleObj)
-        })
-    });
+describe("GET /api", () => {
+  it("should return an object detailing all the available endpoints on the API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((data) => {
+        const sampleObj = allEndpoints;
+        expect(data.body).toEqual(sampleObj);
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  it("should respond with a 200 status code when passed a article id that is correct", () => {
```

Comment:
Since we expect 200 in our other tests down below do we need a separate test for this?